KiwiHC16/Abeille
# Several updates

- Profalux: new gen support C1Z06C+Z10C
- Aqara WS-EUK01 H1 wall switch
- EQ assist update
- EQ page/advanced: display JSON location if custom
- switch branch: do not clean devices_local
- JSON: added model + manufacturer
- check json: enhanced
- Gledopto GL-C-006 prelim
- JSON mainEP fix + added model/manuf
- Removed some obsolete advanced options
